app/model/view/AssignmentSolutionViewFactory.php

```diff
@@ -61,6 +64,11 @@ public function getSolutionData(AssignmentSolution $solution) {
       "bonusPoints" => $solution->getBonusPoints(),
       "lastSubmission" => $solution->getLastSubmission() ? $solution->getLastSubmission()->getData($canViewDetails, $canViewValues) : null,
       "submissions" => $submissions,
+      "commentsStats" => $thread && $user ? [
+        "count" => $this->comments->getThreadCommentsCount($thread, $user),
+        "mineCount" => $this->comments->getThreadCommentsCount($thread, $user, false),
```

Comment:
The name of the column is not as clear as it could be. How about `authoredCount`?

Reply:
OK